test_schema.py

```diff
@@ -0,0 +1,29 @@
+import json
```

Comment:
Not sure if this file is needed.

Reply:
Right. I was testing some things and missed deleting this file. Will remove.